Lib/gftools/scripts/set_overlap_bits.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+from defcon import Font
+from pathops import Path
+from ufo2ft.filters.decomposeComponents import DecomposeComponentsFilter
+from ufo2ft.preProcessor import TTFPreProcessor
+import math
+from fontTools.designspaceLib import DesignSpaceDocument
+
+
+def set_overlap_bits(ufo):
+    # Skip setting bits if ufo
+    if any(g.lib.get("public.truetype.overlap") for g in ufo):
```

Comment:
safe to skip if flags are present (might have been added manually), but perhaps you could also add a CLI flag to overwrite existing overlap flags, as they might have been mechanically generated and out of sync